src/main/java/com/commercetools/emailprocessor/model/Email.java

```diff
@@ -0,0 +1,188 @@
+package com.commercetools.emailprocessor.model;
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.activation.DataSource;
+import javax.mail.internet.InternetAddress;
+import javax.mail.internet.MimeMessage;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.node.TextNode;
+import com.fasterxml.jackson.databind.util.JSONPObject;
+import org.apache.commons.lang3.StringUtils;
+import org.apache.commons.mail.EmailException;
+import org.apache.commons.mail.HtmlEmail;
+import org.apache.commons.mail.util.MimeMessageParser;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+
+public class Email implements Serializable {
+
+    private static final Logger LOG = LoggerFactory.getLogger(Email.class);
+    public static final String PENDING = "pending";
+    public static final String ERROR = "error";
+
+    private String status = "";
+    private String lastError = "";
+
+    private List<InternetAddress> to = null;
+    private List<InternetAddress> cc = null;
+    private List<InternetAddress> bcc = null;
+    private InternetAddress from = null;
+    private String subject = "";
+    private JsonNode body = null;
+    List<String> attachments = new ArrayList();
+
+    public Email() {
+    }
+
+    public Email(HtmlEmail email, String status, String errorMessage) {
+
+        this.status = status;
+        this.lastError = lastError;
+
+        to = email.getToAddresses();
```

Comment:
Why do you need all these attributes?

Reply:
869f0211cb16af04e198ec4fe60b9947d89f0062